### PR TITLE
wip: clean-up uiflex logic

### PR DESCRIPTION
**Pull request by vkozyura**
Merge after cap/cds-mtxs/pull/969 is merged.

### CODE REVIEW

**Comment by patricebender** [comment]
please no internal links on public github :)

File: db-service/lib/cqn2sql.js [hunk 1]
```diff
@@ -494,21 +489,6 @@ class CQN2SQLRenderer {
...
-          const merges = extractions.filter(c => elements?.[c.name]?.['@cds.extension'])
-          if (merges.length) {
-            c.sql = `json_set(ifnull(${c.sql},'{}'),${merges.map(
-              c => this.string('$."' + c.name + '"') + ',' + c.sql,
-            )})`
-          }
-        }
-        return c
-      })
       .filter(a => a)
```
Comment (danjoa) [inline db-service/lib/cqn2sql.js:492]: @BobdenOs can we also remove that filter?

**Review by danjoa — APPROVED** [review]

**Comment by danjoa** [comment]
> Merge after cap/cds-mtxs/pull/969 is merged.

👍 go for it!

File: db-service/lib/cqn2sql.js (hunk 1 above)
Comment (BobdenOs) [reply db-service/lib/cqn2sql.js:492]: The reason for the `filter` is line 500 which is being removed. This change should actually just rename the place where `extraction` is used to `extractions`.

**Pull request merged**